spec/purgatory_spec.rb

```diff
@@ -237,6 +237,30 @@
       end
     end
 
+    context "approving a performable method" do
```

Comment:
Super small thing: Just realized this context should be "approving a performable method that returns false".